src/Language/PureScript/CoreFn/ToJSON.hs

```diff
@@ -64,8 +64,8 @@ moduleToJSON v m = object [ T.pack "imports"   .= map (annToJSON *** moduleNameT
                           ]
 
 bindToJSON :: Bind Ann -> Value
-bindToJSON (NonRec ann n e) = object [ runIdent n .= ( annToJSON ann, exprToJSON e ) ]
-bindToJSON (Rec bs) = object $ map (\((ann, n), e) -> runIdent n .= ( annToJSON ann, exprToJSON e ) ) bs
+bindToJSON (NonRec ann n e) = toJSON [(runIdent n, annToJSON ann, exprToJSON e)]
+bindToJSON (Rec bs) = toJSON $ map (\((ann, n), e) -> (runIdent n, annToJSON ann, exprToJSON e)) bs
```

Comment:
So far I tested this on a [purescript react isomorphic app](https://github.com/coot/purescript-isomorphic-react-example/tree/dce). At the end I run purescript dce-ed code on node and in the browser.

`pure dce` with `uglifyify` produced 705kB while `pulp build -O` with `uglifyify` 685kB.
There is still room for improvement though.  Now I am only removing top level declarations and not touching foreign modules at all.